kroxylicious-operator/src/test/resources/DerivedResourcesTest/virtualcluster-with-tls-with-explict-client-auth-required/in-VirtualKafkaCluster-one.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+kind: VirtualKafkaCluster
+apiVersion: kroxylicious.io/v1alpha1
+metadata:
+  name: one
+  namespace: proxy-ns
+  generation: 4
+spec:
+  proxyRef:
+    name: minimal
+  targetKafkaServiceRef:
+    name: kafka
+  ingresses:
+    - ingressRef:
+        name: cluster-ip
+      tls:
+        certificateRef:
+          name: downstream-tls-cert
+        trustAnchorRef:
+          name: downstream-ca
+          key: cas.pem
+        tlsClientAuthentication: NONE
```

Comment:
test case dir name doesn't align with the configuration (also typo in `explicit`)